Emby.Server.Implementations/AppBase/BaseApplicationPaths.cs

```diff
@@ -12,50 +13,43 @@ public abstract class BaseApplicationPaths : IApplicationPaths
         /// <summary>
         /// Initializes a new instance of the <see cref="BaseApplicationPaths"/> class.
         /// </summary>
-        protected BaseApplicationPaths(
-            string programDataPath,
-            string appFolderPath,
-            string logDirectoryPath = null,
-            string configurationDirectoryPath = null)
+        protected BaseApplicationPaths(string dataDir, string configDir, string logDir)
```

Comment:
Does the function arguments have to change terminology to *Dir instead of *Path? Imho their names were more consistent before

Reply:
My thinking was to match the switch names (--datadir, configdir, logdir).  Plus one could argue dir for folders and path for files..

Reply:
That's a fair point.  I'll change them back.